packages/dd-trace/src/plugins/log_plugin.js

```diff
@@ -45,6 +46,10 @@ module.exports = class LogPlugin extends Plugin {
 
       if (!span) return
 
+      if (this.constructor.name) {
+        span.setTag(COMPONENT, this.constructor.name)
```

Comment:
Logging plugins only copy information from the active span to logs and shouldn't be altering the component of spans.

Reply:
You're correct. I deleted this code change within the log_plugin